### PR TITLE
fix: update to news items

### PR DESCRIPTION
**Pull request by cjakeman**
Additions to the list of new features and improvements in the manual, section 2

### CODE REVIEW

File: Source/Documentation/Manual/news.rst [hunk 1]
```diff
@@ -15,6 +15,7 @@ What's been added
 -----------------
```
Comment (SteelFill) [inline Source/Documentation/Manual/news.rst:15]: Didn't realize I never added a line here with the lighting updates. The new lighting features have proven to be popular with many creators eager to implement them, but I've seen people not realize the features are there until randomly looking through the manual-not ideal! Would help if there was a news entry for awareness there.

Maybe just...
- new rolling stock lighting controls, including horn-activated flashing lights

Comment (cjakeman) [reply Source/Documentation/Manual/news.rst:15]: Thanks, Phillip.

I've added that in now.

Best wishes,

Chris

**Review by Csantucci — APPROVED** [review]
Approved, thanks.

**Comment by cjakeman** [comment]
Thanks, Carlo

**Pull request merged**